src/subcommands/deployments.ts

```diff
@@ -81,6 +119,88 @@ export default async function (args: Args): Promise<void> {
   }
 }
 
+async function listDeployments(args: Args): Promise<void> {
+  if (!args.project) {
+    error(
+      "No project specified. Use --project to specify the project of which to list the deployments",
+    );
+  }
+  const relativeNext = args.next.reduce(
+    (prev, next) => prev + parseInt(next || "1"),
+    0,
+  );
+  const relativePrev = args.prev.reduce(
+    (prev, next) => prev + parseInt(next || "1"),
+    0,
+  );
```

Comment:
What happens if `next === "foo"` for instance? In this case the result of `parseInt` becomes `NaN`, which is probably worth handling maybe